eZ/Publish/API/Repository/Values/User/Policy.php

```diff
@@ -18,7 +18,7 @@
  * @property-read mixed $roleId the role id this policy belongs to
  * @property-read string $module Name of module, associated with the Policy
  * @property-read string $function  Name of the module function Or all functions with '*'
- * @property-read array $limitations an array of \eZ\Publish\API\Repository\Values\User\Limitation
+ * @property-read \eZ\Publish\API\Repository\Values\User\Limitation[] $limitations an array of \eZ\Publish\API\Repository\Values\User\Limitation
```

Comment:
Well, it's only phpdoc, and the commit message isn't linked to this PR. Figured it wouldn't do much harm. But it can be extracted safely of course.